esp/esp/program/modules/forms/teacherreg.py

```diff
@@ -60,6 +60,14 @@ class TeacherClassRegForm(FormWithRequiredCss):
         ("****", "**** - You should not expect to be able to understand most of this class.",),
     ]
 
+    # The following is a dummy list (because using None causes an error). To enable class styles, admins should set the
```

Comment:
Does using `[]` work?  That would be a little cleaner, if possible.  But if not this is fine.